exercises.md

```diff
@@ -101,9 +107,10 @@ Briefly discuss what it means to be important as a group (if you want to)
 #### (10 mins) Together - Discuss
 
 Discuss in the group:
+
 * lines covered by many people?
 * lines named but not by a lot of people
-* Agree less than 8 of the most important lines
+* Agree less than 10 of the most important lines
```

Comment:
(same 8 or 10 question)

Reply:
Again - depends on the length of the code, so facilitator decides what is difficult but not impossible. I'd go with 8.